src/Core/Engine/Physics/PhysicsWorld.cpp

```
@@ -0,0 +1,203 @@
+
+        if (!bodyB.isStatic)
+            bodyB.linearVelocity += impulse * invMassB;
+    }
+}
+
+void PhysicsWorld::SyncTransforms()
+{
+
+}
+
+void PhysicsWorld::HandleAABB(PhysicsBody& PhysBodyA, PhysicsBody& PhysBodyB)
```

Clang (14.0.6)

[warning] method 'HandleAABB' can be made static

(readability-convert-member-functions-to-static)